utils/check_dummies.py

```diff
@@ -103,11 +103,14 @@ def read_init():
 def create_dummy_object(name, backend_name):
     """Create the code for the dummy object corresponding to `name`."""
     _pretrained = [
```

Comment:
This is the list to modify to include new model patterns.

Reply:
Aha, great!